buildfile.m

```diff
@@ -0,0 +1,13 @@
+function plan = buildfile
+
+% Add source code folder to path
```

Comment:
Add the source folder to the path

(Adding "the" for readability)

Q: Why didn't we name the folder source instead of code?